unittests/data/unnamed_classes.hpp

```diff
@@ -25,7 +25,7 @@ struct S1{
 			unsigned int raw;
 		} flags;
 
-		union FileAttribs{
+		union /*FileAttribs*/{
```

Comment:
I wanted to make sure enums without tag names were also detected correctly by the type trait, and since the `Flags` union above and `FileAttribs` here were otherwise not testing different functionality, I repurposed this definition for different test.